test/Service/AuthorizationServiceTest.php

```diff
@@ -157,15 +160,15 @@ public function testReturnsTrueForIdentityWhenHasPermissionButNoAssertionsExists
 
     public function testUsesAssertionsAsInstances()
     {
-        $role = new FlatRole('admin');
-        $identity = new Identity([$role]);
+        $role      = new FlatRole('admin');
```

Comment:
@danizord not sure why it hurts but if so then I can suggest you this:
```
$role      = new FlatRole       ('admin');
$identity  = new Identity       ([$role]);
$assertion = new SimpleAssertion();
```
just joking but maybe not :D